Tasks/JavaToolInstallerV0/javatoolinstaller.ts

```diff
@@ -81,35 +85,129 @@ async function getJava(versionSpec: string) {
     toolLib.prependPath(path.join(jdkDirectory, BIN_FOLDER));
 }
 
-function sleepFor(sleepDurationInMillisecondsSeconds): Promise<any> {
-    return new Promise((resolve, reeject) => {
-        setTimeout(resolve, sleepDurationInMillisecondsSeconds);
-    });
+/**
+ * Return file ending if it is supported. Otherwise throw an error.
+ * @param file Path to a file.
+ */
+function getSupportedFileEnding(file: string): string {
```

Comment:
Could you shortly describe in js doc - what file ending is? And what is the difference between file ending and file extension?